src/controllers/requestController.ts

```diff
@@ -76,6 +78,13 @@ export class RequestController {
     }
 
     private async runCore(httpRequest: HttpRequest) {
+        if (undefined !== httpRequest.confirmSendMsg) {
+            const userConfirmed = await window.showQuickPick(['Yes', 'No'], { canPickMany: false, placeHolder: httpRequest.confirmSendMsg });

```

Comment:
Is better to use the `window.showErrorMessage` with a modal option like following:
```js
const result = await window.showErrorMessage('Are you sure to send the request', { modal: true }, 'Yes');
if (result !== 'Yes') {
    return;
}
```